### DISCUSSION TITLE
Slow inkscape, thoughts on optimising work flow?

### DISCUSSION THREAD

**Discussion by Skyclemo** [ExperienceLovelace/ha-floorplan#322] [General]
Good day everyone, 

This is super fun, kind of got addicted to Ha floorplan. Was wondering if anyone can point me to the right direction on these areas:

1. Very very slow Inkscape.
The file size is 20kb only. BUT it contains mesh and gradients. 
My laptop is a dell i7 deforce ftx, should be able to handle

a. Is there a good alternative to inkscape? I tried to use adobe Illustrator but it does not have xml editor in built.
b. Any way to optimize inkscape for floorplan work?  Here's a link for ideas 
https://inkscape.org/learn/tutorials/avoid-performance-issues/
c. Any way to copy and past SVG code into inkscape GUI editor directly? My current workflow is either to open both svg files and copy the layers. OR open both SVG in visual studios and copy code.
d. Any thoughts on best universal dimensions for the common phones? My current version is wayyy to short.

Sorry for the many questions, do let me know if any other info is required. Many thanks

<img width="511" alt="ww1" src="https://user-images.githubusercontent.com/103242092/213980415-4dd150c3-6415-4718-8ca5-f4001eccaa8f.png">

**Comment by OzGav** [comment]
Not sure. My main SVG is 5249kB and I dont have any issues? There must be something else going on...

**Reply by exetico** [reply]
I've not faced any perfomance issues, too. @Skyclemo - It's impossible to give you other inputs, without the source file.
